src/engine/source/kvdb/src/kvdbManager.cpp

```diff
@@ -75,6 +80,7 @@ void KVDBManager::unloadDB(const std::string& name)
     if (isLoaded)
     {
         m_dbs.erase(name);
+        Metrics::instance().addUpDownCounterValue("Kvdb.DatabaseInUseCounter", -1L);
```

Comment:
nitpick: Could you put a brief comment above the metric to know what it measures?